eoscostarica/eos-rate
# Define color palette for radars

### Use a defined color palette for BPs radars.

As specified in #170, let's use a defined color palette for the radars, each BP gets a random selected color from that palette.

Remove some unused files as well.

### Steps to test

1. Navigate to the block producers page.
2. Check that the new colors are showing up in the radars.

### Checklist
